sympy/printing/latex.py

```diff
@@ -699,6 +700,8 @@ def _hprint_Function(self, func):
         func = self._deal_with_super_sub(func)
         if func in accepted_latex_functions:
             name = r"\%s" % func
+            if self._settings["ln_notation"] and name == r"\log":
```

Comment:
how about:
```python
if self._settings["ln_notation"] and func == 'log':
    name = r"\ln"
else:
    name = r"\%s" % func
```

Reply:
This logic should be added as a new `_print_log` method.